Simple solvers/AD.py

```diff
@@ -2,6 +2,16 @@
 
 #have user enter a,b,c to solve quadratic 
```

Comment:
This comment is outdated since you changed the prompts... please update to reflect what the program is doing

Reply:
Please update and push your changes.